src/MassTransit.Abstractions/JobService/Contracts/JobService/JobPriority.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace MassTransit.Contracts.JobService
+{
+    public enum JobPriority
+    {
+        Low = 0,
+        BelowNormal = 1,
+        Normal = 2,
+        AboveNormal = 3,
+        High = 4
```

Comment:
```suggestion
        Lowest = 0,
        Low = 1,
        Normal = 2,
        High = 3,
        Highest = 4
```

Reply:
GPT alternatives:
```
    Priority 0: Background
    Priority 1: Low
    Priority 2: Standard
    Priority 3: Urgent
    Priority 4: Critical

    Priority 0: Non-critical
    Priority 1: Routine
    Priority 2: Important
    Priority 3: Critical
    Priority 4: Emergency

    Priority 0: Optional
    Priority 1: Minor
    Priority 2: Normal
    Priority 3: Major
    Priority 4: Critical

    Priority 0: Deferred
    Priority 1: Pending
    Priority 2: Actionable
    Priority 3: Critical
    Priority 4: Immediate
`

Reply:
`JobPriority.Optional`: Maybe we run it or maybe we just forget about it 😆 

I think I like your suggestion best. It is to the point and it is quite clear what the hierarchy is between them.